src/actions/authActions.jsx

```diff
@@ -136,3 +136,134 @@ export const googleAuth = () => {
       });
   };
 };
+
+export const updateProfile = (profileInfo) => {
+  return (dispatch, getState, { getFirebase, getFirestore }) => {
+    const firestore = getFirestore();
+    firestore
+      .collection("profiles")
+      .doc(profileInfo.userId)
+      .update(profileInfo)
+      .then(() => {
+        dispatch({ type: "UPDATE_SUCCESS" });
```

Comment:
not sure if this just depends on the other PR but these should be constant variables too as you changed in the other PR

Reply:
Actually this is for another PR because I already made these changes